pulp-core/patches/content_disposition_update.patch

```diff
@@ -1,8 +1,8 @@
 diff --git a/pulpcore/app/settings.py b/pulpcore/app/settings.py
-index aa65d01b6..dcce9a55f 100644
+index 8a544803a..45c1ea547 100644
 --- a/pulpcore/app/settings.py
 +++ b/pulpcore/app/settings.py
-@@ -55,6 +55,12 @@ FILE_UPLOAD_HANDLERS = ("pulpcore.app.files.HashingFileUploadHandler",)
```

Comment:
The patch within a patch broke my brain for a second. 😅

Reply:
diffception